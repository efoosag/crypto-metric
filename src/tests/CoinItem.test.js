/* eslint-disable import/no-named-as-default */
/* eslint-disable comma-dangle */
import React from 'react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Coinitems from '../components/Home';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Coinitems />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
