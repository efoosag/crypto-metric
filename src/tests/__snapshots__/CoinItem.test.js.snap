// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    className="search"
  >
    <br />
    <input
      className="search"
      onChange={[Function]}
      placeholder="SEARCH CRYPTO COIN"
      type="text"
      value=""
    />
  </form>
  <div
    className="cryptoCase"
  >
     
  </div>
</div>
`;
